### PR TITLE
set: add Polish translation

### PR DESCRIPTION
**Pull request by DawidStruzik**
- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.pl/windows/set.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+> Wyświetl lub ustaw zmienne środowiskowe dla bieżącej instancji CMD.
+> Więcej informacji: <https://learn.microsoft.com/windows-server/administration/windows-commands/set>.
+
+- Lista wszystkich bieżących zmiennych środowiskowych:
+
+`set`
+
+- Ustaw zmienną środowiskową na określoną wartość:
+
+`set {{name}}={{value}}`
```
Comment (kbdharun) [inline pages.pl/windows/set.md:12]: Can you translate `{{name}}` and `{{value}}`?

File: pages.pl/windows/set.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+- Wymień zmienne środowiskowe zaczynające się od podanego ciągu znaków:
+
+`set {{name}}`
```
Comment (kbdharun) [inline pages.pl/windows/set.md:16]: Translate `{{name}}` here too

File: pages.pl/windows/set.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+- Pyta użytkownika o wartość dla określonej zmiennej:
+
+`set /p {{name}}={{prompt_string}}`
```
Comment (kbdharun) [inline pages.pl/windows/set.md:20]: Also here, can you translate the 2 arguments?

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have made some suggestions for this page.

> We generally prefer to translate the command arguments inside `{{}}` in translation pages.

File: pages.pl/windows/set.md (hunk 1 above)
Comment (DawidStruzik) [reply pages.pl/windows/set.md]: The values have been updated

File: pages.pl/windows/set.md (hunk 3 above)
Comment (DawidStruzik) [reply pages.pl/windows/set.md]: The values have been updated

File: pages.pl/windows/set.md (hunk 2 above)
Comment (DawidStruzik) [reply pages.pl/windows/set.md]: The value has been updated

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**